Foundation/NSURLSession/http/HTTPURLProtocol.swift

```diff
@@ -495,6 +498,16 @@ extension _HTTPURLProtocol: _EasyHandleDelegate {
         }
     }
 
+    fileprivate func notifyDelegate(aboutSentData count: Int64) {
+        guard let session = self.task?.session as? URLSession else { fatalError() }
```

Comment:
maybe a better error message in the `fatalError()`? Otherwise just `as! URLSession`